kafka_consumer/tests/test_e2e.py

```diff
@@ -0,0 +1,13 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from tests.common import assert_check_kafka
```

Comment:
For testing it's best to not rely on the current working directory:

```suggestion
from .common import assert_check_kafka
```